### PR TITLE
chore(codes): update clients for token code experiment

### PR DESCRIPTION
**Pull request by vbudhram**
Connects to https://github.com/mozilla/fxa-content-server/issues/5990

Reopened https://github.com/mozilla/fxa-content-server/pull/6429 against train 120. It enables token code for Lockbox and only Mozilla/Softvision emails.

### CODE REVIEW

**Comment by vbudhram** [comment]
@mozilla/fxa-devs r?

File: app/scripts/lib/experiments/grouping-rules/token-code.js [hunk 1]
```diff
@@ -31,14 +38,24 @@ module.exports = class TokenCodeGroupingRule extends BaseGroupingRule {
   }
 
   choose(subject) {
-    if (! subject || ! subject.uniqueUserId || ! subject.experimentGroupingRules || ! subject.isTokenCodeSupported) {
+    if (! subject || ! subject.uniqueUserId || ! subject.experimentGroupingRules || ! subject.isTokenCodeSupported || ! subject.account) {
```
Comment (shane-tomlinson) [inline app/scripts/lib/experiments/grouping-rules/token-code.js]: I don't see `experimentGroupingRules` used anywhere, is it needed?

File: app/scripts/lib/experiments/grouping-rules/token-code.js [hunk 2]
```diff
@@ -6,17 +6,24 @@
...
-  '3a1f53aabe17ba32': {
-    name: 'Firefox Add-ons',
-    rolloutRate: 0.0 // Rollout rate between 0..1
-  },
-  'dcdb5ae7add825d2': {
-    name: '123Done',
+  '37fdfa37698f251a': {
+    enableTestEmails: false,
+    groups: GROUPS_DEFAULT,
+    name: 'Lockbox Extension',
```
Comment (shane-tomlinson) [inline app/scripts/lib/experiments/grouping-rules/token-code.js]: Is the name just to help us as devs? I don't see it used anywhere.

File: app/scripts/lib/experiments/grouping-rules/token-code.js [hunk 3]
```diff
@@ -6,17 +6,24 @@
...
     rolloutRate: 0.0
   },
+  '98adfa37698f255b': {
+    enableTestEmails: true,
+    groups: ['treatment-code'],
+    name: 'Lockbox Extension iOS',
```
Comment (shane-tomlinson) [inline app/scripts/lib/experiments/grouping-rules/token-code.js]: Extension?

File: app/scripts/lib/experiments/grouping-rules/token-code.js (hunk 2 above)
Comment (vbudhram) [reply app/scripts/lib/experiments/grouping-rules/token-code.js]: Correct, name is just used for devs.

File: app/scripts/lib/experiments/grouping-rules/token-code.js (hunk 3 above)
Comment (vbudhram) [reply app/scripts/lib/experiments/grouping-rules/token-code.js]: This is the Lockbox app on iOS.

**Pull request merged**